app/uk/gov/hmrc/tradergoodsprofilesdatastore/connectors/RouterConnector.scala

```diff
@@ -204,4 +209,18 @@ class RouterConnector @Inject() (config: Configuration, httpClient: HttpClientV2
           case _        => Future.failed(UpstreamErrorResponse(response.body, response.status))
         }
       }
+
+  def getEmail(
```

Comment:
Might be missing something. Isn't this duplicate of CustomsDataStoreConnector?

Reply:
Yep - will sort!